ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -167,7 +167,7 @@ public func toRACSignal<Value: AnyObject, Error: NSError>(signal: Signal<Value?,
                 subscriber.sendError(error)
             case .Completed:
                 subscriber.sendCompleted()
-            default:
+			case .Interrupted:
```

Comment:
Tabs vs. Spaces.
The old line was indented wit a total of 12 spaces, the new line is indented with 3 tabs.